index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>Form</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+  <div class="top">
+    <h1>freeCodeCamp Survey Form</h1>
+    <p><em>Thank you for taking the time to help us improve the platform</em></p>
+  </div>
+
```

Comment:
Use semantic tags instead of divs. Replace the div with a header tag for this section

Reply:
- [ ] Great one with this feedback, but I think it is lacking a more description of the semantic tags.
Kindly remind them what semantic tags are by adding an external link here because it will make your review more understandable to the author of the code and will make your review more concise to foster quick understanding.

Reply:
- [ ] Great one with this feedback, but I think it is lacking a more description of the semantic tags.
Kindly remind them what semantic tags are by adding an external link here because it will make your review more understandable to the author of the code and will make your review more concise to foster quick understanding.

Reply:
For the top section, you can use semantic tags to better explain what that part of the code holds. For instance in this section, you can use the header tag. Please refer to [this ](https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md)article for more information.